src/components/app.tsx

```diff
@@ -120,8 +140,22 @@ export class AppComponent extends BaseComponent<IProps, IState> {
 
         </Simulation>
 
+        <DatGui data={simulationOptions} onUpdate={this.handleUpdate}>
+          <DatButton label="Model options" onClick={this.toggleShowOptions} />
+          { showOptionsDialog &&
+            [
+              <DatBoolean path="requireEruption" label="Require eruption?" key="requireEruption" />,
+              <DatBoolean path="requirePainting" label="Require painting?" key="requirePainting" />
+            ]
+          }
+        </DatGui>
+
       </App>
     );
   }
 
+  private toggleShowOptions = () => this.setState({showOptionsDialog: !this.state.showOptionsDialog});
+
+  private handleUpdate = (simulationOptions: any) => this.setState({ simulationOptions });
```

Comment:
Seems like eventually we could use `interface SimulationAuthoringOptions` for the  type  here.

Reply:
True! I'll put that in whatever PR comes next, so as not to need an extra rebase on the Maps work.